docs/content/preview/yugabyte-voyager/migrate/live-migrate.md

```diff
@@ -512,6 +513,12 @@ The export directory has the following sub-directories and files:
 - `metainfo` and `temp` directories are used by yb-voyager for internal bookkeeping.
 - `logs` directory contains the log files for each command.
 
+## Assess Migration
+
+This step is optional and only required for PostgreSQL and Orcale Database Migration. Assess migration assess the migration complexity and get schema changes, data distribution and cluster sizing recommendations using the `yb-voyager assess` command.
```

Comment:
The command is `yb-voyager assess-migration` . Pls take care of all instances where `yb-voyager assess` is mentioned.